Project3_STL/headers/Triangle.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include "Point3D.h"
+
+namespace Shapes3D
+{
+    class Triangle1

```

Comment:
Class name should be Triangle, not Triangle1

Reply:
> Class name should be Triangle, not Triangle1

I had a compiler issue yesterday so was trying with Vinayak if renaming classes might sort things out. I will rename Triangle class.